release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1207,6 +1207,16 @@ In the following tables, features are marked with the following statuses:
 
 [id="ocp-telco-core-4-16-known-issues_{context}"]
 
+* When a worker node's Topology Manager policy is changed, the NUMA-aware secondary pod scheduler does not respect this change, which can result in incorrect scheduling decisions and unexpected topology affinity errors. As a workaround, restart the NUMA-aware scheduler by deleting the NUMA-aware scheduler pod. (link:https://issues.redhat.com/browse/OCPBUGS-34583[*OCPBUGS-34583*])
+
+* Due to an issue with Kubernetes, the CPU Manager is unable to return CPU resources from the last pod admitted to a node to the pool of available CPU resources. These resources are allocatable if a subsequent pod is admitted to the node. However, this in turn becomes the last pod, and again, the CPU manager cannot return this pod's resources to the available pool.
```

Comment:
Is there a bug to link to?

Reply:
I find this one a bit confusing, and a touch convoluted. But, I have no idea how to reword it. 

```suggestion
* Due to an issue with Kubernetes, the CPU Manager is unable to return to the pool of available CPU resources the resources from the last pod admitted to a node. The resources from the last pod are reallocatable if a subsequent pod is admitted to the node. However, this pod becomes the last pod, and again, the CPU manager cannot return this pod's resources to the available pool.
```